File: pracazplikami.py
with open('Tekst1.txt', 'r') as tekst1, open('Tekst2.txt', 'r') as tekst2, open('suma.txt', 'w') as tekst3:
    for line in tekst1:
        tekst3.write(line)
    tekst3.write("\n")
    for line in tekst2:
        tekst3.write(line)

File: pracazplikami.cpp
#include <iostream>
#include <fstream>
using namespace std;

ifstream tekst1("Tekst1.txt");
ifstream tekst2("Tekst2.txt");
ofstream tekst3("suma.txt");

int main() 
{
  string a;
  while(tekst1.good())
  {
    getline(tekst1, a);
    tekst3 << a << endl;
  }
  tekst1.close();
  while(tekst2.good())
  {
    getline(tekst2, a);
    tekst3 << a << endl;
  }
  tekst2.close();
}